src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -23,4 +23,12 @@ export class DefaultTaskService implements TaskService {
   getAll(): Observable<Task[]> {
     return this.http.get<Task[]>(this.baseUrl + '/tasks');
   }
+
+  updateStatus(id: string, status: string): Observable<void> {
+    return this.http.get<void>(this.baseUrl + '/tasks/' + id);
```

Comment:
you wouldn't update the status with a get

Reply:
:) Missed that. Updated with put.